### PR TITLE
Add support for package manager selection

### PR DESCRIPTION
**Pull request by DarkoKukovec**
Adds a `--package-manager` flag that can currently be set to `yarn`. Might add some other options in future if needed. Related to #21.

Currently has no effect on squawk, but will be used once #26 is done.

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ const loglevel = argv.loglevel || 'info';
 const logger = getLogger(loglevel);
 const options = { loglevel: loglevel };
 
-runner(argv.webpack, argv.dependency, options)
+runner(argv.webpack, argv.dependency, options, argv.packageManager)
```
Comment (andreicek) [inline lib/index.js]: I would place this in `options`. With the addition of #23, this is going to get complicated fast.

File: test_modules/package.json [hunk 2]
```diff
@@ -1,4 +1,5 @@
 {
   "name": "webpack-canary-test",
-  "version": "0.0.1"
+  "version": "0.0.1",
```
Comment (andreicek) [inline test_modules/package.json:3]: Is there a reason why is this needed in `test_modules`?

**Review by andreicek — CHANGES_REQUESTED** [review]

File: test_modules/package.json (hunk 2 above)
Comment (DarkoKukovec) [reply test_modules/package.json:3]: What specifically? The file or the info in it?
The file is needed so that npm knows it should install modules here, and not in the parent folder.
The info is not technically required, but it prevents some  yarn warnings. This could probably be removed once "Support yarn warnings in stderr" from #31 is implemented.

File: lib/index.js (hunk 1 above)
Comment (DarkoKukovec) [reply lib/index.js]: Agree, fixed.

File: test_modules/package.json (hunk 2 above)
Comment (andreicek) [reply test_modules/package.json:3]: Ok!

**Review by andreicek — APPROVED** [review]
